src/main/bash/copy-images.sh

```diff
@@ -1,27 +1,43 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 DRY_RUN=false; [ "$1" = "--dry-run" ] && DRY_RUN=true && shift
 
 version="$1"
 registry="$2"
-curl -o versions.yaml https://raw.githubusercontent.com/entando-k8s/entando-k8s-controller-coordinator/v${version}/charts/entando-k8s-controller-coordinator/templates/docker-image-info-configmap.yaml
+curl -f -o versions.yaml https://raw.githubusercontent.com/entando-k8s/entando-k8s-controller-coordinator/v${version}/charts/entando-k8s-controller-coordinator/templates/docker-image-info-configmap.yaml
+retcode=$?
+if [ $retcode -ne 0 ]; then
```

Comment:
@ampie wonder if you are aware that this is not going to catch http error statuses but only network errors

you need something line this for that purpose

```
res="$(curl -f -o versions.yaml https: -o versions.yaml -w "%{http_code}" ..)"
if [ "$res" ...
```

Reply:
The manual says: "-f, --fail          Fail silently (no output at all) on HTTP errors"
I have actually been testing that locally and it fails against 404 errors because the tag wasn't in Github yet.

Reply:
yep, you are right